plugin/heroku-nodejs-plugin-node-21.sha512

```diff
@@ -0,0 +1 @@
+107fa5474f01fce59c2b7e9d17411f161feddd1a0537ac85798ae8804d66ebe17285d66062adfe75472b64006951b81dcc322e4918e2ee5c6f3fae96c24dd1e6  heroku-nodejs-plugin-node-21-v13.tar.gz
```

Comment:
Not sure about these files in the plugin directory, they need to be removed probably